apps/ewallet_db/lib/ewallet_db/mint.ex

```diff
@@ -15,7 +15,7 @@ defmodule EWalletDB.Mint do
     external_id(prefix: "mnt_")
 
     field(:description, :string)
-    field(:amount, EWalletDB.Types.Integer)
+    field(:amount, EWalletConfig.Types.Integer)
```

Comment:
Should types be part of `EWalletDB` than `EWalletConfig`? Especially when it's used to define the schema's field type.

Reply:
I don't think it should, and this would actually allow us to clean up the shared ewallet_db / local_ledger_db stuff. The config is lower than the db apps and can be used by any other sub app.